livesync/indico_livesync/handler.py

```diff
@@ -88,12 +79,17 @@ def connect_signals(plugin):
     plugin.connect(signals.attachments.attachment_updated, _attachment_changed)
 
 
-def _moved(obj, old_parent, new_parent, **kwargs):
+def _moved_event(obj, old_category, **kwargs):
```

Comment:
I'd say that since the convention with event handlers right now is to name the function after the event, we should keep this one consistent.
In this particular case, it would have to be `_register_move`, since this handler is also used by categories. Anyway, prefixing everything with `_register` seems like a bit of an overkill to me...